src/pages/commits/ActiveCommits/buildVariantCard/FailedTaskStatusIcon.tsx

```diff
@@ -0,0 +1,78 @@
+import { useLazyQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import IconButton from "@leafygreen-ui/icon-button";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Skeleton } from "antd";
+import { Link } from "react-router-dom";
+import { TaskStatusIcon } from "components/TaskStatusIcon";
+import { getTaskRoute } from "constants/routes";
+import {
+  GetFailedTaskStatusIconTooltipQuery,
+  GetFailedTaskStatusIconTooltipQueryVariables,
+} from "gql/generated/types";
+import { GET_FAILED_TASK_STATUS_ICON_TOOLTIP } from "gql/queries";
+import { msToDuration } from "utils/string";
+
+interface FailedStatusIconProps {
+  taskId: string;
+  status: string;
+}
+export const FailedTaskStatusIcon: React.FC<FailedStatusIconProps> = ({
+  taskId,
+  status,
+}) => {
+  const [loadData, { data, loading }] = useLazyQuery<
+    GetFailedTaskStatusIconTooltipQuery,
+    GetFailedTaskStatusIconTooltipQueryVariables
+  >(GET_FAILED_TASK_STATUS_ICON_TOOLTIP, { variables: { taskId } });
+  const { displayName, timeTaken } = data?.task ?? {};
+  const { testResults } = data?.taskTests ?? {};
+
+  return (
+    <Tooltip
+      usePortal={false}
+      align="top"
+      justify="middle"
+      popoverZIndex={1}
+      trigger={
+        <div
+          onMouseOver={() => loadData()}
+          onFocus={() => loadData()}
+          data-cy="failed-task-status-icon"
+        >
+          <Link
+            data-cy="task-status-icon"
+            to={getTaskRoute(taskId)}
+            key={`task_${taskId}`}
+          >
+            <IconButton aria-label="task icon">
+              <TaskStatusIcon status={status} size={16} />
+            </IconButton>
+          </Link>
+        </div>
+      }
+      triggerEvent="hover"
+    >
+      <div data-cy="failed-task-status-icon-tooltip">
+        {loading ? (
+          <Skeleton />
+        ) : (
+          !!data && (
+            <>
+              <TooltipTitle data-cy="failed-task-status-icon-tooltip-title">
+                {displayName} - {msToDuration(timeTaken)}
+              </TooltipTitle>
+              {testResults?.map(({ id, testFile }) => (
+                <div key={id}>{testFile}</div>
```

Comment:
Can we apply some sort of wrap or max width to the tooltip to avoid this cutoff? ([URL using prod data](http://localhost:3000/commits/mms?skipOrderNumber=32192&statuses=failed-umbrella,failed,task-timed-out,test-timed-out,known-issue,success,running-umbrella,started,dispatched))
<img width="1552" alt="image" src="https://user-images.githubusercontent.com/9298431/137177521-e4af3056-8d80-4354-9e49-d3967d72b529.png">